packages/core/bootstrap/src/lib/metrics/util.ts

```diff
@@ -65,51 +65,58 @@ function buildSymbolString(data: string | string[]): string {
  * @returns {string}
  */
 export const getFeedId = (input: AdapterRequest): string => {
-  const commonFeedParams = {
-    base: ['base', 'from', 'coin', 'symbol', 'asset'],
-    quote: ['quote', 'to', 'convert'],
-  }
-
-  // check if string is within array
-  function includesCheck(param: string) {
-    return Object.keys(input.data).includes(param)
-  }
-
-  /**
-   * If the request is coming from the cache warmer, use a fixed id. This is to reduce the
-   * cardinality of the feed_id label in prometheus, which can be overloaded quickly
-   */
-  if (input.debug?.warmer) {
-    return WARMER_FEED_ID
-  }
-
-  // run through validator if input.data object has keys that match potential base and quote parameters
-  if (commonFeedParams.base.some(includesCheck) && commonFeedParams.quote.some(includesCheck)) {
-    const validationResult = new Validator(input, commonFeedParams)
-    if (validationResult.error) {
-      logger.debug('Unable to validate feed name')
-      return JSON.stringify(input)
+  try {
+    const commonFeedParams = {
+      base: ['base', 'from', 'coin', 'symbol', 'asset'],
+      quote: ['quote', 'to', 'convert'],
     }
 
-    const { base, quote } = validationResult.validated.data
+    // check if string is within array
+    const includesCheck = (param: string) => Object.keys(input.data).includes(param)
 
     /**
-     * With batched requests, the base can either be an array of bases, or a single base.
-     * The same type constraints apply to the quote param.
+     * If the request is coming from the cache warmer, use a fixed id. This is to reduce the
+     * cardinality of the feed_id label in prometheus, which can be overloaded quickly
      */
-    if (base) {
-      return `${buildSymbolString(base)}` + (quote ? `/${buildSymbolString(quote)}` : '')
+    if (input.debug?.warmer) {
+      return WARMER_FEED_ID
     }
-  }
-
-  const entries = Object.keys(input)
-    .filter((prop) => !excludableAdapterRequestProperties[prop])
-    .map((k) => [k, input[k as keyof AdapterRequest]])
 
-  const rawFeedId = JSON.stringify(Object.fromEntries(entries))
+    // run through validator if input.data object has keys that match potential base and quote parameters
+    if (commonFeedParams.base.some(includesCheck) && commonFeedParams.quote.some(includesCheck)) {
+      const validationResult = new Validator(input, commonFeedParams)
+      if (validationResult.error) {
+        logger.debug('Unable to validate feed name')
```

Comment:
could be good to log out the param being checked here as well

Reply:
The `logger` here now looks like:
```
logger.debug('Unable to validate feed name', {
          input,
          error: validationResult.error.toString(),
        })
```